DAO section added (#30)
klaytn/klaytn-metaverse-package-docs: docs/klaytn-contracts/dao-contracts/governance-tools.md

---
title: ⛓ Governance Tools
sidebar_label: Governance Tools
---

> This page contains the list of governance tools to help you build decentralized organizations(DAO) on Klaytn.

Reviewer: ```suggestion
This page contains the list of governance tools which can be integrated with the dapp to build 
a DAO in Metaverse.
```
